static/js/containers/pages/CheckoutPage.js

```diff
@@ -140,16 +146,26 @@ export class CheckoutPage extends React.Component<Props, State> {
     try {
       const basketResponse = await updateBasket(basketPayload)
       if (basketResponse.status !== 200) {
-        if (basketResponse.body.errors) {
+        if (basketResponse.body && basketResponse.body.errors) {
           actions.setErrors(basketResponse.body.errors)
+        } else {
+          Sentry.captureException(
+            new Error(`Basket Response status: ${basketResponse.status}`)
+          )
+          actions.setErrors({ items: GENERIC_ERROR_MESSAGE })
         }
```

Comment:
We should do report the error in both branches of the if statement. Also, we should include the entire `basketResponse` in the reported message, not just the status code. Also, since this is repeated 3 times can we make a helper function in the file to handle this error reporting?

Reply:
Done.

Reply:
Description updated with screenshots of additional info being passed to Sentry now.